src/js/common/schemaform/definitions/dateRange.js

```diff
@@ -3,17 +3,23 @@ import * as date from './date';
 
 import commonDefinitions from 'vets-json-schema/dist/definitions.json';
 
-export const schema = commonDefinitions.dateRange;
+export const schema = {
+  type: 'object',
+  properties: {
+    from: commonDefinitions.date,
+    to: commonDefinitions.date
+  }
+};
```

Comment:
@raquelromano-gov similar conversation started [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/659) if you're interested in starting that!